infra/scripts/create_omop_cdm_synapse.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+LOG_FILE=${AZ_SCRIPTS_PATH_OUTPUT_DIRECTORY}/all.log
+
+exec >  >(tee -ia "${LOG_FILE}")
+exec 2> >(tee -ia "${LOG_FILE}" >&2)
+
+apk --update add gettext gnupg
+
+# Download the desired package(s)
```

Comment:
Can we provide a URL to download those packages in case we need to upgrade the driver's version?

I believe it was taken from: https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver16&tabs=alpine18-install%2Calpine17-install%2Cdebian8-install%2Credhat7-13-install%2Crhel7-offline